### ISSUE TITLE
[P3.1][Sourcing][Accessibility][ATL testing][Multi-input]: "X more" button has no visible focus in mullti-input control

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10623]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
"X more" button has no visible focus in mullti-input control

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
ng: 15
platform: 0.43.x

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Select many tokens until user sees, "X more" blue color link , try to focus and see there is no visible focus on it.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
reported by ATL

**Comment by droshev** [comment]
this issue is reporting the opposite of what is stated [here](https://github.com/SAP/fundamental-ngx/issues/10620)

**Comment by I543348** [comment]
@sagar7259 pls help if this issue can be closed?

**Comment by sagar7259** [comment]
@droshev this issue talks about the keyboard focus which is not visible on the UI and it is not related to the screen reader issue rather its a keyboard related issue.

Here on navigating to the 'x more' button, there is no visible focus provided. But when this was accessed using VPC mode of screen reader then we go to know that this element is read has a link and not a button, hence a different ticket was reported for the screen reader issue.

Both issues are different and valid.

Regards,
Sagar

**Comment by I543348** [comment]
Hi NGX team ,
pls note its about a visible border issue , on focus can be fixed by css styling.
nothing related to screenreader has been reported in this issue and hence its not conflicting with https://github.com/SAP/fundamental-ngx/issues/10620

cc: @droshev , @sagar7259

**Comment by mikerodonnell89** [comment]
Closing this issue as keyboard focus on this button has been removed by https://github.com/SAP/fundamental-ngx/pull/10653

**Comment by sagar7259** [comment]
Hi Team,
The keyboard focus has been removed now for the 'x more' link and this solves the issue by automatically reading the total number of tokens along with the label and UI type of multi input when accessed. 
This is working as expected.
Therefore this ticket can be closed.

Regards,
Sagar
